packages/gafl-webapp-service/src/pages/renewals/identify/route.js

```diff
@@ -30,11 +30,7 @@ const getData = async request => {
 const schema = Joi.object({
   referenceNumber: validation.permission.permissionNumberUniqueComponentValidator(Joi),
   'date-of-birth': validation.contact.createBirthDateValidator(Joi),
-  postcode: Joi.alternatives().conditional('country-code', {
-    is: Joi.string().pattern(/^GB/),
-    then: validation.contact.createUKPostcodeValidator(Joi),
-    otherwise: validation.contact.createOverseasPostcodeValidator(Joi)
-  })
+  postcode: validation.contact.createOverseasPostcodeValidator(Joi)
```

Comment:
Was there a problem with the way that it was deciding which validation to do? Could we have a quick chat about this?